test.js

```diff
@@ -1,148 +1,203 @@
-        outplan.create("foo", ["A", "B"]);
-        var value = outplan.expose("foo", 1);
-        assert(value === "A" || value === "B", value);
-    });
-    
-    it("deterministically gets values", function() {
-        outplan.create("foo", ["A", "B"]);
-        assert.equal(outplan.expose("foo", 1), "B");
-        assert.equal(outplan.expose("foo", 2), "B");
-        assert.equal(outplan.expose("foo", 3), "A");
-        assert.equal(outplan.expose("foo", 4), "B");
-        assert.equal(outplan.expose("foo", "1"), "B");
-        assert.equal(outplan.expose("foo", "2"), "B");
-        assert.equal(outplan.expose("foo", "3"), "A");
-        assert.equal(outplan.expose("foo", "4"), "B");
-        assert.equal(outplan.expose("foo", "5"), "B");
-        assert.equal(outplan.expose("foo", "6"), "A");
-        assert.equal(outplan.expose("foo", "7"), "A");
-        assert.equal(outplan.expose("foo", "8"), "B");
-        assert.equal(outplan.expose("foo", "9"), "A");
+        testBoth(function(outplan) {
+            outplan.create("foo", ["A", "B"]);
+            var value = outplan.expose("foo", 1);
+            assert(value === "A" || value === "B", value);
+        });
+    });
+    
+    it("deterministically gets values in outplan_full", function() {
+        outplanFull.create("foo", ["A", "B"]);
+        assert.equal(outplanFull.expose("foo", 1), "B");
+        assert.equal(outplanFull.expose("foo", 2), "B");
+        assert.equal(outplanFull.expose("foo", 3), "A");
+        assert.equal(outplanFull.expose("foo", 4), "B");
+        assert.equal(outplanFull.expose("foo", "1"), "B");
+        assert.equal(outplanFull.expose("foo", "2"), "B");
+        assert.equal(outplanFull.expose("foo", "3"), "A");
+        assert.equal(outplanFull.expose("foo", "4"), "B");
+        assert.equal(outplanFull.expose("foo", "5"), "B");
+        assert.equal(outplanFull.expose("foo", "6"), "A");
+        assert.equal(outplanFull.expose("foo", "7"), "A");
+        assert.equal(outplanFull.expose("foo", "8"), "B");
+        assert.equal(outplanFull.expose("foo", "9"), "A");
+    });
+    
+    it("deterministically gets values in lite version", function() {
+        outplanLite.create("foo", ["A", "B"]);
+        assert.equal(outplanLite.expose("foo", 1), "A");
+        assert.equal(outplanLite.expose("foo", 2), "B");
+        assert.equal(outplanLite.expose("foo", 3), "A");
+        assert.equal(outplanLite.expose("foo", 4), "B");
+        assert.equal(outplanLite.expose("foo", "1"), "A");
+        assert.equal(outplanLite.expose("foo", "2"), "B");
+        assert.equal(outplanLite.expose("foo", "3"), "A");
+        assert.equal(outplanLite.expose("foo", "4"), "B");
+        assert.equal(outplanLite.expose("foo", "5"), "A");
+        assert.equal(outplanLite.expose("foo", "6"), "B");
+        assert.equal(outplanLite.expose("foo", "7"), "A");
+        assert.equal(outplanLite.expose("foo", "8"), "B");
+        assert.equal(outplanLite.expose("foo", "9"), "A");
```

Comment:
Unfortunately the results above are no good. `string-hash` gives very random hashes like `215288988744`, `244818780217` but unfortunately the last digit is too much dependent on the input giving these very predictable A/B/A/B results.